azkaban-spi/src/main/java/azkaban/spi/StorageMetadata.java

```diff
@@ -23,31 +23,31 @@
 
 
 public class StorageMetadata {
-  private final String projectId;
-  private final String version;
-  private final String extension;
+  private final int projectId;
+  private final int version;
+  private final String uploader;
```

Comment:
It is a hack to get `DatabaseStorage` to work without too much trouble. It may not be required later on. However, adding more metadata / context should hurt the code.